SGVHAK_Rover/templates/chassis_config.html

```diff
@@ -26,49 +26,74 @@
 <script type="text/javascript" src="{{url_for('static', filename='chassis_config.js')}}"></script>
 <input type="hidden" id="request_wheel_status" value="{{url_for('request_wheel_status')}}" />
 <div class="container">
-  <h4 class="center_header">Chassis Configuraton</h4>
-  <table>
+  
+  <!-- <table> -->
   {% for wheelTableRow in wheelTable.values() %}
-    <tr>
-      {% for wheelTableColumn in wheelTableRow.values() %}
-      <td class="center-align">
+    
+    <!-- <tr> -->
+    <div class="row">
+      <!-- {% for wheelTableColumn in wheelTableRow.values() %} -->
+     
+      <!-- <td class="center-align"> -->
+      
         {% for wheel in wheelTableColumn %}
-        <div class="container blue lighten-4">
-          <canvas id="canvas_{{wheel['name']}}" width="100" height="100">
-          </canvas>
-          <br/>
-          <b>{{wheel['name']}}</b>
-          <br/>
-          ({{wheel['x']}},{{wheel['y']}})
-          <br/>
-          {% if wheel['rolling'] == None %}
-            <b>Freewheel</b>
-          {% else %}
-            <b>Rolling:</b> M{{wheel['rolling'][1]}}@{{wheel['rolling'][0]}}
-            {% if wheel['rolling'][2] == True %}
-              (Inverted)
-            {% endif %}
-            <br/>
-            {{roboclaw_table[wheel['name']][0]}}
-          {% endif %}
-          <br/>
-          <b>Steering:</b>
-          {% if wheel['steering'] == None %}
-            Fixed
-          {% else %}
-            M{{wheel['steering'][1]}}@{{wheel['steering'][0]}}
-            {% if wheel['steering'][2] == True %}
-              (Inverted)
-            {% endif %}
-            <br/>
-            {{roboclaw_table[wheel['name']][1]}}
-          {% endif %}
-        </div>
-        {% endfor %}
-      </td>
-      {% endfor %}
-    </tr>
+  
+          
+            <div class="col s6 m4 {{wheelOffset[wheel['name']]}}">
```

Comment:
This block is indented too far. It should only be one indent beyond its immediate HTML parent. (This `<div class="col s6 m4 [...]` should only be two spaces beyond `<div class="row">`